docs/src/_tutorials/jupyter.md

```diff
@@ -0,0 +1,177 @@
+---
+title: How to use jupyter with Meltano
+description: Using jupyter Python notebook as transformation tooling or to run analyses across your meltano maintained data.
+layout: doc
+weight: 4
+---
+
+As of _Sep-2022_, jupyter is not yet available as utility on the [hub](https://hub.meltano.com/), so you need to add it to your Meltano project as (local) custom plugin. Even if it is available as plugin on the hub, you will probably want to do some customization, so it will make sense to create your own version.
+
+The steps involved are:
+1. Add a (local) custom jupyter utility
+2. Add potential Python libraries you will need
+3. _Optional: Add database connection variables to be exposed into the environment_
+4. Add Papermill or nbconvert to execute your data transformations & customize commands for jupyter to run the notebook
+5. Execute your notebooks & add them to a schedule
+
+_Steps 1-4 are customizing the meltano.yml to suit your setup, if you're comfortable with the process, read through them, and go straight for step 4 to copy & customize the yaml block from there, and you're ready to go._
+
+## 1. Add a (local) custom jupyter utility
+You can add a custom plugin either [via the CLI](https://docs.meltano.com/guide/plugin-management#custom-plugins) or [using the YAML file](https://docs.meltano.com/concepts/project#custom-plugin-definitions). As jupyter serves multiple purposes, the type "utility" is recommended as plugin type.
+
+For jupyter, you can choose both the "classi notebook" installed via the pip-package "jupyter" or the newer jupyterlab installed via the pip-package jupyterlab.
```

Comment:
```suggestion
For Jupyter, you can choose both the "classic notebook" installed via the pip-package "jupyter" or the newer jupyterlab installed via the pip-package jupyterlab.
```

Lots of jupyter in this sentence. I feel like a link out to the sources could help here as well.